src/main/java/mate/academy/spring/model/dto/response/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.spring.model.dto.response;
+
+import java.util.List;
+import mate.academy.spring.model.User;
+
+public class ShoppingCartResponseDto {
+    private Long id;
+    private List<Long> ticketIds;
+    private User user;
+
```

Comment:
```suggestion
    private Long id;
    private List<Long> ticketIds;

```

as shoppingCard id is the same as user id, and we don't need to return entity(user) in response dto